ui/shared/components/panel/family.jsx

```diff
@@ -46,6 +50,53 @@ const mapSampleDispatchToProps = (state, ownProps) => ({
 
 const FirstSample = connect(mapSampleDispatchToProps)(BaseFirstSample)
 
+const AnalystEmailDropdown = ({ load, loading, usersByUsername, onChange, value, ...props }) =>
+  <DataLoader load={load} loading={false} content>
+    <AddableSelect
```

Comment:
I don't think this should be an `AddableSelect` I think it should be a regular select, because we only want them to be able to choose an existing user and not add a new one